src/containers/Guide/GuideIntroduction/index.tsx

```diff
@@ -38,18 +38,52 @@ const GuideIntroduction: FC = () => {
-          quantified solely by human judgment.
+          This documentation outlines the trust-based distributed architecture of our network. The architecture offers
+          an efficient and scalable peer-to-peer consensus mechanism using the election of a primary validation node by
+          a distributed network of trusted nodes. The amount of trust in the network is quantified solely by human
+          judgment. Here is why.
         </p>
+        <DocSubHeader>Enabling humans to do what they do best</DocSubHeader>
         <p>
-          The architecture is based on the idea that when building a distributed payment ledger, it is not the
-          transaction processing itself that requires distribution across multiple servers. This often results in
-          duplicate work being done by several network nodes, causing an inherent inefficiency in the system. It is
-          rather the ability to elect fairly a single validation server and consensual acceptance of the produced
-          results that require distribution among peers. This allows for highly performant transaction validation within
-          a decentralized network.
+          There is no formula or function that judges human value in the way which we truly need to. Computers are great
+          at solving questions where there is only one definitive answer, but to determine and accurately judge
+          something as intangible as the work produced by a human being, they are still a long way off.
+        </p>
+        <DocSubHeader>Incentives</DocSubHeader>
+        <p>
+          This leaves us with the challenge of humans themselves. Humans, unlike computer algorithms, can be corrupted.
+          However, because of how we want to reward our community members, we believe that there is no better system
+          than human judgement for quantifying a wide variety of work (knowing that many jobs are still unknown).
+          Choosing a government who truly has the best interest of the community at heart can ensure prosperity, as it
+          happens in actual life, in enterprises, institutions, organizations, or countries around the globe. No matter
+          how many checks and balances we put in place, no matter how many restrictions or regulations we write, we will
+          be always getting back to the variable of human judgement.
+        </p>
+        <DocSubHeader>Value and work put into the community</DocSubHeader>
+        <p>
+          We want to ensure that everyone (not just developers and DevOps) can receive the value of their work put into
+          making our network and community better. In the long run, this is the only way to guarantee success for our
+          community as a group of people working toward common goals. In fact, we see thenewboston as an everlasting
+          project because only work put into it generates value spread across every unit of our cryptocurrency. Every
```

Comment:
```suggestion
          project because only work put into it generates value.  Every
```

Reply:
I am ok with this update, not technical, marketing people know better here :)

Reply:
Denis, can you suggested change does not pass npm lint. Not sure how to revert this now